src/Fossa/API/Types.hs

```diff
@@ -40,6 +40,9 @@ data ApiOpts = ApiOpts
   }
   deriving (Eq, Ord, Show)
 
+instance ToJSON ApiOpts where
+  toJSON opts = object ["uri" .= show (apiOptsUri opts)]
```

Comment:
[suggestion]: include `"apikey" .= "<REDACTED>"`, so it's clear to the reader that this is where the key is stored.